docs/ref/templates/builtins.txt

```diff
@@ -1732,13 +1732,30 @@ example, when the active locale is ``en`` (English):
 ``34232.00`` ``{{ value|floatformat:"-3g" }}`` ``34,232``
 ============ ================================= =============
 
+Output is always localized (independently of the :ttag:`{% localize off %}
+<localize>` tag) unless the argument passed to ``floatformat`` has the ``u``
+suffix, it will force disabling localization. For example, when the active
+locale is ``pl`` (Polish):
+
+============ ================================= =============
+``value``    Template                          Output
+============ ================================= =============
+``34.23234`` ``{{ value|floatformat:"2" }}``   ``34,232``
+``34.23234`` ``{{ value|floatformat:"2u" }}``  ``34.232``
```

Comment:
`Output` column seems to probably have been based on the prior example, where the argument was `3` not `2` decimal places?

Reply:
Good catch :dart: